hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java

```diff
@@ -81,21 +87,19 @@ public CypherModel submitQuery(String cypherQuery,@Nullable Map<String, String>
 
     private RequestMessage createRequest(String cypherQuery) {
         return RequestMessage.build(Tokens.OPS_EVAL)
-                .processor("cypher")
-                .add(Tokens.ARGS_GREMLIN, cypherQuery)
-                .create();
+                             .processor("cypher")
+                             .add(Tokens.ARGS_GREMLIN, cypherQuery)
+                             .create();
     }
 
     private List<Object> doQueryList(Client client, RequestMessage request)
-            throws ExecutionException, InterruptedException {
-
-        ResultSet results = null;
-        results = client.submitAsync(request).get();
+        throws ExecutionException, InterruptedException {
+        ResultSet results = client.submitAsync(request).get();
 
         Iterator<Result> iter = results.iterator();
         List<Object> list = new LinkedList<>();
 
-        for (; iter.hasNext(); ) {
+        while (iter.hasNext()) {
```

Comment:
prefer to use ArrayList instead of LinkedList

Reply:
It's better to use a LinkedList in two conditions which are itemized below:
1. The amount of items is unknown.
2. There is no need to access items by index.

Reply:
LinkedList is not a memory-friendly structure, it generates a lot of extra objects.
And in most scenarios, the performance of ArrayList is better, you can try to test them.